core/templates/dev/head/pages/exploration_editor/settings_tab/settings_tab.html

```diff
@@ -286,11 +286,12 @@ <h3>Roles</h3>
                   </span>
                 </div>
 
-                <input type="submit" class="btn btn-default protractor-test-save-role" value="Save">
+                <input type="submit" class="btn btn-default protractor-test-save-role" value="Save" ng-disabled="hasTitleNotBeenGiven()">
                 <button type="button" class="btn btn-default" ng-click="closeEditRolesForm()"
                         ng-show="isRolesFormOpen">
                   Cancel
                 </button>
+                <p ng-show="hasTitleNotBeenGiven()" class="text-danger">*Title is needed in order to invite users</p>
```

Comment:
```suggestion
                <p ng-show="hasTitleNotBeenGiven()" class="text-danger">Please provide a title before inviting.</p>
```

Reply:
done

Reply:
Doesn't look done. Please confirm you made changes before saying Done. Thanks!

Reply:
Slipped by (Saw it showed outdated so did not recheck i will be more careful)
Sorry I will be more careful.